Tags work

mitkoyanev opened the pull request:
can now add and remove tags.

mitkoyanev committed:
Tags work

can now add and remove tags.

mitkoyanev committed:
added a space between tags

added a space between tags

mitkoyanev committed:
ading editor

sheelasathyanarayana committed:
Merge branch 'master' into feature/story-creation-component

mitkoyanev committed:
edited out the broken thing

mitkoyanev committed:
Merge branch 'master' into feature/story-creation-component

mitkoyanev committed:
Selected tags text changes now.

mitkoyanev committed:
no change

mitkoyanev committed:
Merge branch 'feature/story-creation-component' of https://github.com/TruthTreeASD/cassiopeia into feature/story-creation-component

mitkoyanev committed:
no change

sheelasathyanarayana reviewed (commented):
Clicking on create story tab displayed a blank page in the deployment

mitkoyanev committed:
Merge branch 'master' into feature/story-creation-component

mitkoyanev committed:
broken editor

froala editor component is broken. may need jquery

mitkoyanev committed:
added react-quill

updated story field and local state
Now store both HTML of rich text, and the text itself. Ready for updating with AIP!

mitkoyanev commented:
Closed issue
